projects/WhatsAppClone/src/client.ts

```diff
@@ -0,0 +1,11 @@
+import {StreamChat} from 'stream-chat'
+import {STREAM_API_KEY, STREAM_USER_TOKEN, STREAM_USER_ID} from '@env'
+import {ChatContextValue} from 'stream-chat-react-native'
+
+export const chatClient = StreamChat.getInstance(
+  STREAM_API_KEY,
+) as unknown as ChatContextValue['client']
```

Comment:
Look into why this is needed

Reply:
Our `Chat` component expects a `StreamChat<StreamChatGenerics>` type under and it just does not match with what `getInstance` gives back 
```
StreamChat<
    AttachmentType,
    ChannelType,
    CommandType,
    EventType,
    MessageType,
    ReactionType,
    UserType
  >
```

Reply:
I will create an issue for the SDK